lighthouse-treemap/app/styles/treemap.css

```diff
@@ -230,3 +234,140 @@ header {
   text-align: center;
   word-break: break-word;
 }
+
+/* Copied from viewer.css */
+
+.drop_zone {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  visibility: hidden;
+}
+.drop_zone.dropping {
+  visibility: visible;
+  font-size: var(--heading-font-size);
+  background-color: rgba(255,255,255,0.8);
+  color: var(--unknown-color);
+}
+.drop_zone.dropping::after {
+  content: 'Drop report here';
+  border: 2px dashed currentColor;
+  border-radius: 5px;
+  padding: 25px;
+  width: 33vw;
+  min-width: 250px;
+  height: 20vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fff;
+}
+.treemap-placeholder {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #555;
+  font-weight: 300;
+}
+.treemap-placeholder-inner {
+  display: flex;
+  align-items: center;
+  border-radius: 5px;
+  padding: 16px;
+  max-width: 80vw;
+  border: 2px dashed rgba(0,0,0,0.2);
+  cursor: pointer;
+  background-color: #fff;
+}
+.treemap-placeholder-inner.lh-loading {
+  filter: blur(2px) grayscale(1);
+  cursor: wait
+}
+.treemap-placeholder-inner.lh-loading>div {
+  pointer-events: none;
+}
+.treemap-placeholder-inner.dropping {
+  border-color: currentColor;
+}
+.treemap-placeholder__heading {
+  font-weight: 300;
+  margin: 0;
+  line-height: 32px;
+}
+.treemap-placeholder__help {
+  margin-top: 12px;
+  line-height: 1.6;
+}
+.treemap-placeholder-logo {
+  width: 140px;
+  height: 140px;
+}
+.treemap-placeholder__url {
+  padding: 8px;
+  width: 100%;
+  border: 1px solid #eee;
+  margin-top: 16px;
+  display: none;
+}
+
+
+@media screen and (max-width: 635px) {
+  .treemap-placeholder-inner {
+    display: block;
+    text-align: center;
+  }
+  .treemap-placeholder-logo {
+    width: 100px;
+    height: 100px;
+  }
+  .treemap-placeholder__url {
+    display: block;
+  }
+}
+
+@media screen and (min-width: 636px) {
+  .treemap-placeholder-inner {
+    padding: 40px 32px;
+  }
+  .treemap-placeholder-logo {
+    margin-right: 16px;
+  }
+}
+
+/* app z-indexes */
+.drop_zone {
+  z-index: 3;
+}
+
+/* Copied from report-styles.css */
+
+#lh-log {
+  position: fixed;
+  background-color: #323232;
+  color: #fff;
+  min-height: 48px;
+  min-width: 288px;
+  padding: 16px 24px;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
+  border-radius: 2px;
+  margin: 12px;
+  font-size: 14px;
+  cursor: default;
+  transition: transform 0.3s, opacity 0.3s;
+  transform: translateY(100px);
+  opacity: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 3;
```

Comment:
```suggestion
  z-index: 3;
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
```

looked weird without this
![image](https://user-images.githubusercontent.com/2301202/119003914-bb41ae00-b953-11eb-89a1-b16c03cb6489.png)